pom.xml

```diff
@@ -160,30 +141,6 @@
             <version>2.15.2</version>
         </dependency>
 
-        <dependency>
-            <groupId>org.apache.maven</groupId>
-            <artifactId>maven-model</artifactId>
-            <version>3.3.9</version>
-            <scope>provided</scope>
-        </dependency>
-        <dependency>
-            <groupId>org.apache.maven</groupId>
-            <artifactId>maven-plugin-api</artifactId>
-            <version>3.3.9</version>
-            <scope>provided</scope>
-        </dependency>
-        
-        <dependency>
-            <groupId>org.jenkins-ci.main</groupId>
-            <artifactId>jenkins-test-harness</artifactId>
-            <scope>test</scope>
-        </dependency>
-        <dependency>
-            <groupId>org.apache.maven.plugins</groupId>
-            <artifactId>maven-failsafe-plugin</artifactId>
```

Comment:
Will this affect integration tests?

Reply:
I didn't run the integration tests completely, but we can invoke `mvn failsafe:integration-test` or `mvn integration-test` even without this.

Besides, if we need to specify a fixed version of `maven-failsafe-plugin`, we need to add a `plugin` definition in the `build` section, rather than in the `dependencies` section.